app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -256,9 +256,17 @@ function handleCommentSubmit(event) {
             var mainCommentsForm = document.getElementById("new_comment");
             mainCommentsForm.classList.remove("submitting");
             mainCommentsForm.classList.remove('preview-open');
-            const textArea = form.getElementsByClassName("comment-textarea")[0];
-            textArea.closest('.comment-form').classList.remove('comment-form--initiated');
-            textArea.value = newComment.comment_template || "";
+
+            const commentInputs = [...form.getElementsByClassName("comment-textarea")]
```

Comment:
Still not certain if this is the best approach @aitchiss. I was wondering if we can only render one of the textareas instead of both in the `<MentionAutoComplete />` component. I did a quick test, but it looks like it causes rendering issues with layout shifts and fun stuff.

Reply:
Yeah I see what you mean, it feels wrong for this code to have to care about the detail of the autocomplete implementation 🤔 

One thought is that we only need to make sure the value is applied in the 'plain' (non combobox) text area - this is presented at all time the autosuggest isn't in action, and the autosuggest can't be triggered without some kind of `onChange` event, which makes sure it stays in step with whatever the plain text area's current value is.

We could adjust the selector to `form.querySelector('.comment-textarea:not([role=combobox])');` - this will yield only the plain text area for updating and would save having to loop over multiple elements.

Reply:
I'll try that out. Thanks for the suggestion. I think part of the issue might also be that since the attributes are copied, the duplicate textarea is a required field as well. I'll report back. 😅

Reply:
gah! so many things coming out of the woodwork 😅 I guess we could always override the combobox `required` attribute so it's always `false` 🤔 The 'plain' textarea should be the main source of truth

Reply:
Actually here looping through all to clear is not that bad. It's below in the `validateField` function that it's potentially an issue. I'll take it for a test drive.